CONTRIBUTING.rst

```diff
@@ -0,0 +1,157 @@
+
+.. tip::
+    Before you open an issue, check all the existing issues for the DevPortal to make sure you're not duplicating it. 
+
+
+**Contributing via pull requests (PR)**
+
+If you have your own article to add or you want to modify an existing article yourself, here's what you do:
+
+1. Fork the repo.
+2. Make your update.
+3. Open a pull request (make sure you use a helpful PR and commit message). 
+4. Submit your PR, and get it reviewed (one reviewer is required). 
+
+We'll merge it!
+
+.. note::
+    Our build runs several checks on your updates - from linting to spell check to broken links to help you commit good content. Look at the **Checks** section in your pull request to make sure everything passes. If anything fails, check the error details, or reach out to us, and we'll help you. 
+
+
+Style tips
+----------
+
+If you're writing content, here are some high-level tips that can be useful:
+
+- Decide what type of content you want to contribute (check the content model above), and use our content type templates - for concept, task, reference and tutorial.
+
+- Write a good title (start with a verb for tasks, and noun for concepts). 
+
+- Note down your main ideas, and decide what formatting will best support these (use the content templates):
+    - Ordered lists - for tasks
+    - Paragraphs - for definitions and explanations
+    - Lists and tables - for reference/options
+
+- Highlight information by using the following elements:
+    - Tips - for interesting facts about parts of your content/alternative ways of doing something, and shortcuts
+    - Warnings - for important facts readers need to know before proceeding
+    - Info - for any extra information (this may be skipped)
+
+- Add good links (internal and external) - properly formatted, to reputable sources of information.
+
+- Choose your visuals (images, diagrams, screenshots) and include them in a specified size (no bigger than…). 
+
+
+
+Style guides and other resources
+================================
+
+- RestrecturedText cheatsheet
+
+Templates for DevPortal content (by type)
+-----------------------------------------
+
+**Concept**::
+
+    ---
+    title: *About [subject]* (if this is a background information for a task, e.g. *About migrating to Aiven*) / *Subject* (use noun or noun phrase, e.g. *Authentication*, *High availability*)
+    short description: Answer the question "What is this?" and "Why do I care about this?" If the concept is unfamiliar, start with a brief definition.
+    ---
+
+    ## A section here
+
+    <!-- Write one or two paragraphs about the main idea of your topic, as a summary. -->
+    <!-- Make sure you don't have any content that isn't preceded by a header, or it won't be linkable in our TOC. -->
+
+    ## Another section here
+
+    <!-- Write one or two paragraphs about another element of your topic. -->
+    <!-- Keep adding headers and sections until you've completed your article.  -->
+
```

Comment:
Templates should be in restructuredtext format so that people can use it as a basis for their contributions (edit: the same comment for all the templates)